pkg/storaged/details.js

```diff
@@ -34,7 +34,25 @@ define([
      */
 
     function init_details(client, jobs) {
-        var type, name;
+        var type, name, state;
+
+        function ensure_state_multipathd_running() {
```

Comment:
The word `ensure` means that `multipathd` will be running after this. How about something like `check_state_multipathd` or `poll_multipathd_running_state`?